src/main/java/ratpack/zipkin/ServerTracingModule.java

```diff
@@ -156,16 +156,14 @@ public Config spanReporterV2(final Reporter<Span> reporter) {
       return this;
     }
 
-    /** @deprecated please use {@link #spanReporterV2(Reporter)}
+    /**
+     * Set the V1 Span reporter.
      *
      * @param reporter a V1 reporter
      *
      * @return the config
-     * */
-    // Until this is removed, we need a mandatory dep on io.zipkin.reporter:zipkin-reporter
-    // due to overloading requiring access to all overloaded types
-    @Deprecated
-    public Config spanReporter(final zipkin.reporter.Reporter<zipkin.Span> reporter) {
+     */
+    public Config spanReporterV1(final zipkin.reporter.Reporter<zipkin.Span> reporter) {
```

Comment:
@llinder any thoughts on leaving v1 support in for longer? As I mentioned in #40 before merging, I think it might be more user-friendly to leave v1 support in as long as possible...